### PR TITLE
Spades 3.5.0

### PR DESCRIPTION
**Pull request by pjbriggs**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

PR adding a legacy version of `SPAdes` (version 3.5.0).

(NB although MacOS binaries appear to be available for this version, MacOS support has been excluded as the CircleCI builds for this platform stall for unknown reasons and have to be killed. As this is a legacy version of the software it doesn't seem worth expending effort to investigate at this time.)

### CODE REVIEW

File: recipes/spades/3.5.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  url: http://spades.bioinf.spbau.ru/release3.5.0/SPAdes-3.5.0-Linux.tar.gz  # [linux]
+  md5: 1a4ef6e9c9da8894353fb76ed1a31c9d                                      # [linux]
+
+build:
+  number: 0
+  skip: True  # [py36 or osx]
+
+requirements:
+  run:
+    - python <=3.3
```
Comment (bgruening) [inline recipes/spades/3.5.0/meta.yaml]: Can you remove this pinning and adjust the one above at build/skip?

**Comment by pjbriggs** [comment]
Hi @bgruening thanks for the review - I've updated, and while the skip looks a bit ugly it seems to work. Does this seem okay to you?

File: recipes/spades/3.5.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  version: '3.5.0'
+
+source:
+  fn: SPAdes-3.5.0-Linux.tar.gz                                              # [linux]
+  url: http://spades.bioinf.spbau.ru/release3.5.0/SPAdes-3.5.0-Linux.tar.gz  # [linux]
+  md5: 1a4ef6e9c9da8894353fb76ed1a31c9d                                      # [linux]
+
+build:
+  number: 0
+  skip: True  # [py34 or py35 or py36 or osx]
```
Comment (bgruening) [inline recipes/spades/3.5.0/meta.yaml]: [py3k or osx] would be more correct isn't it?

File: recipes/spades/3.5.0/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package:
+  name: spades
+  version: '3.5.0'
+
+source:
+  fn: SPAdes-3.5.0-Linux.tar.gz                                              # [linux]
```
Comment (bgruening) [inline recipes/spades/3.5.0/meta.yaml]: if you skip osx you don't need to pin this here to linux

File: recipes/spades/3.5.0/meta.yaml (hunk 2 above)
Comment (pjbriggs) [reply recipes/spades/3.5.0/meta.yaml]: This version of SPAdes can be built with Python 3.3 (hence the original pinning), which does appear to be supported by conda. Using `[py3k or ...]` would presumably disable the package for Python 3.3 within bioconda.
Is there a neater way to maintain support for Python 3.3? Or should it just drop support for all Python3 versions? (It's not needed for the pipeline I'm trying to support.)

Comment (bgruening) [reply recipes/spades/3.5.0/meta.yaml]: We do not build for 3.3 anymore. See here: https://github.com/bioconda/bioconda-recipes/blob/master/scripts/env_matrix.yml#L1

Comment (pjbriggs) [reply recipes/spades/3.5.0/meta.yaml]: Thanks - it's as you say then!

**Comment by bgruening** [comment]
Cool thanks Peter!

**Comment by pjbriggs** [comment]
Thanks Bjoern!

**Pull request merged**